pages/login.vue

```diff
@@ -1,10 +1,14 @@
 <template>
   <v-card>
     <v-form>
+      <div align="center">
```

Comment:
should remove the `align="center"`. Please add a css/sass section to the component for css changes.

#### How do I do this in sass?
You can update the `<style scoped>` section to be `<style scope lang="scss">`
you can then add any css rules you want. If there's a class that you would normally add for styling, like `mb-md-5`, then use ` @extend` like this

```
<div class="login-container">
    <h1 id="page-title">{{ $t("login.adminLogin") }}</h1>
...

<style scope lang="scss">
    .login-container {
        flexbox css here for centering content
    }
    #page-title {
        @extend .mb-md-5
    }
...
</style>
```

Reply:
Thanks, I'll try to change this
FYI I got this from https://github.com/ourjapanlife/findadoc-frontend/blob/897f4489adf56d31ddd3f2f1f1e2e9e54803ecf7/pages/index.vue#L3

Reply:
- removed `align="center"`
- added style to the heading as following:
```
.login-fields {
  text-align: center;
}
```